server/views/components/event-promo/event-promo.njk

```diff
@@ -46,5 +39,11 @@
         <time datetime="{{ model.start }}">{{ model.start | formatTime }}</time>&mdash;<time datetime="{{ model.end }}">{{ model.end | formatTime }}</time>
       </p>
 
+      {# TODO: add e.g. 'drop in'/'enquire to book'/'ticketed' #}
```

Comment:
I don't understand the TODO? This does that?
Not "Drop in" though as that's separate.

Reply:
Think that was your comment, but you're right I'll take it out